app/controllers/concerns/foreman/controller/csv_responder.rb

```diff
@@ -1,10 +1,11 @@
 module Foreman::Controller::CsvResponder
   extend ActiveSupport::Concern
 
-  def csv_response(resources, columns = csv_columns, header = nil)
+  def csv_response(resources, columns = csv_columns, header = nil, filename = nil)
```

Comment:
@johnpmitsch I am not sure what other plugins are using this csv response method. I am very hesitant to change the method signature here. @ohadlevy @dLobatog @ShimShtein @tbrisker ^ What do you guys think.

Reply:
if its fairly straightforward to change here and in katello, and no other plugins use csv_responder, I think we should consider doing so, but I understand your concern.

I'm just trying to avoid the pattern of multiple default args being added to a method because you can't override them individually in certain cases, i.e. in this PR how would I just override just filename or header?

Named arguments make that easy:
```ruby
def csv_repsonse(resource, columns: csv_columns, header: nil, filename: nil)
end

csv_response(resource, {filename: "foo"})